test/squid.js

```diff
@@ -218,6 +267,69 @@ describe('Squid', () => {
         objectIds.tokenId = registerTxn.events[0].parsedJson.token_id.id;
     }
 
+    function getSwapData() {
+        const swap1 = bcsStructs.squid.DeepbookV3SwapData.serialize({
+            swap_type: { DeepbookV3: null },
+            pool_id: pools.ab,
+            has_base: true,
+            min_output: 1,
+            base_type: coins.a.type.slice(2),
+            quote_type: coins.b.type.slice(2),
+            lot_size: 100,
+            should_sweep: true,
+        }).toBytes();
+        const swap2 = bcsStructs.squid.DeepbookV3SwapData.serialize({
```

Comment:
```suggestion

        const swap2 = bcsStructs.squid.DeepbookV3SwapData.serialize({
```
separate these with newlines